.github/workflows/preflight.yml

```diff
@@ -41,9 +42,11 @@ jobs:
         env:
           FLY_PREFLIGHT_TEST_ACCESS_TOKEN: ${{ secrets.FLYCTL_PREFLIGHT_CI_FLY_API_TOKEN }}
           FLY_PREFLIGHT_TEST_FLY_ORG: flyctl-ci-preflight
-          FLY_PREFLIGHT_TEST_FLY_REGIONS: iad syd
+          # This VM size is only available in ORD.
+          FLY_PREFLIGHT_TEST_FLY_REGIONS: ord
```

Comment:
some tests relies on the secondary region to be set. Is this disabling those preflight tests forever?